apps/backend/app/services/feature_builder_service.py

```
@@ -0,0 +1,691 @@
+        return await self.delete(
+            resource_id=feature_id,
+            user_id=user_id,
+            soft_delete=False
+        )
+
+    async def apply_feature_to_dataset(
+        self,
+        feature_id: str,
+        user_id: str,
+        output_column_name: Optional[str] = None,
+        create_new_dataset: bool = False
```

Ruff (0.14.10)

Unused method argument: `create_new_dataset`

(ARG002)